ops/docker/Dockerfile

```diff
@@ -14,15 +16,15 @@ COPY packages/message-relayer/package.json ./packages/message-relayer/package.js
 COPY packages/replica-healthcheck/package.json ./packages/replica-healthcheck/package.json
 COPY packages/regenesis-surgery/package.json ./packages/regenesis-surgery/package.json
 COPY integration-tests/package.json ./integration-tests/package.json
-RUN yarn install --frozen-lockfile
+RUN yarn install
```

Comment:
Just checked the file, previously this was removed because I only changed the node version of batch submitter and DTL, so we got two different node versions and`--frozen-lockfile` was causing some compatibility issues for me because of two different node versions. But later on after discussed with Aiden, I've changed all node versions to 18, which we will not have conflicts on node versions, so this can be added back.